### DISCUSSION TITLE
Using multiple filters on one update request

### DISCUSSION THREAD

**Discussion by HaMM4R** [supabase/supabase#1152] [Questions] [answered]
Hi all, 

I'm new to using supabase and REST API's as a whole and just wondered if there was a way to perform multiple .eq filters on one update request? I'm building a leaderboard in Godot and it needs to modify values based on two rows being the same rather than one. Any help would be much appreciated :)

**Comment by inian** [comment, marked as answer]
Hey @HaMM4R,

Yup you can chain multiple .eq filters in the same query. Here is an [example](https://github.com/vercel/nextjs-subscription-payments/blob/5953c9ae34f0b2e7ba368a4a671dabb1c78cd2f3/utils/supabase-client.js#L12).

**Reply by HaMM4R** [reply]
Hi inian, thanks for the quick response, it must just be the Godot plugin for supabase that I'm using doesn't support it yet.

**Reply by saltcod** [reply]
Hey guys! Curious about the syntax for multiple fields — I want to filter years. Is it .`.select().eq('year', '2017').eq('year', '2018').eq('year', '2019')`. or is there some kind of array syntax there? 
thanks!

**Reply by steve-chavez** [reply]
@saltcod An [in](https://supabase.io/docs/reference/javascript/in) filter(`.in('year', ['2017', '2018', '2019'])`) would be fit there.

**Reply by saltcod** [reply]
Missed this reply — THANK YOU! Amazing!
